src/Sulu/Bundle/ContentBundle/Resources/public/js/components/ckeditor-internal-link/main.js

```diff
@@ -212,7 +212,7 @@ define(['underscore', 'text!./form.html'], function(_, formTemplate) {
 
                                 this.href = id;
                                 this.hrefTitle = title;
-                                this.hrefPublished = item.publishedState;
+                                this.hrefPublished = !!item.published;
```

Comment:
This seems to work initially, but when reloading the page, the same error still persists.